### PR TITLE
feat: replace dummy contact info with org info

### PR DESCRIPTION
**Pull request by sean-dickinson**
Adds the organization contact information to the footer partial

# 🔗 Issue
https://github.com/rubyforgood/pet-rescue/issues/171

# ✍️ Description
- Replaces the dummy contact info in the footer with the org's contact info

# 📷 Screenshots/Demos
image

### CODE REVIEW

File: test/system/home_page_test.rb [hunk 1]
```diff
@@ -8,9 +8,17 @@ class HomePageTest < ApplicationSystemTestCase
   end
 
   test "renders custom hero and about text from PageText or default text" do
-    PageText.create(hero: "Super Pets for a good paws", about: "All about us")
+    create(:page_text, hero: "Super Pets for a good paws", about: "All about us")
+
     visit home_index_path
     assert_text "Super Pets for a good paws"
     assert_text "All about us"
+
```
Comment (sean-dickinson) [inline test/system/home_page_test.rb:16]: I wasn't sure if this was the best place to test this. Happy to put it elsewhere if there's a better place for it.

File: app/views/layouts/shared/_footer.html.erb [hunk 2]
```diff
@@ -76,9 +76,9 @@
         <!-- contact info -->
         <div class="mb-4">
           <h3 class="fw-bold mb-3">Get in touch</h3>
-          <p>339 McDermott Points Hettingerhaven, NV 15283</p>
-          <p class="mb-1">Email: <a href="#">support@geeksui.com</a></p>
-          <p>Phone: <span class="text-dark fw-semibold">(000) 123 456 789</span></p>
+          <p><%= Current.tenant.location.city_town %>, <%= Current.tenant.location.province_state %> <%= Current.tenant.location.country %></p>
+          <p class="mb-1">Email: <a href="mailto:<%= Current.tenant.profile.email %>"><%= Current.tenant.profile.email %></a></p>
+          <p>Phone: <span class="text-dark fw-semibold"><%= Current.tenant.profile.phone_number %></span></p>
```
Comment (jmilljr24) [inline app/views/layouts/shared/_footer.html.erb:81]: What are your thoughts on formatting the phone number? 

```suggestion
          <p>Phone: <span class="text-dark fw-semibold"><%= Current.tenant.profile.formatted_phone %></span></p>
```

Screenshot from 2024-05-01 19-36-28

We have a method in the `adopter_foster_profile` model you could reference and add to `organization_profile` model

**Review by jmilljr24 — CHANGES_REQUESTED** [review]
Thanks for working on this! It looks good. I think we can go without testing for this unless some else thinks its prudent. 

Do you think you could have the phone number formatted to make it a bit easier to read? We use the Phonelib gem which can do most of the work. Let me know what you think!!

File: test/system/home_page_test.rb (hunk 1 above)
Comment (sean-dickinson) [reply test/system/home_page_test.rb:16]: Yeah that makes sense, but I always worry about adding extra cases for a system test just given how slow they are comparatively. I think you're right that there's not much benefit here

File: app/views/layouts/shared/_footer.html.erb (hunk 2 above)
Comment (sean-dickinson) [reply app/views/layouts/shared/_footer.html.erb:81]: Sounds good, I'll take a look and pull it into a concern for reuse

Comment (sean-dickinson) [reply app/views/layouts/shared/_footer.html.erb]: Hey @jmilljr24 I went ahead and refactored this into a concern. I included the `before_save` normalizing in there as well. Let me know what you think.

**Comment by jmilljr24** [comment]
Awesome job! Thanks @sean-dickinson 

Can you please run `rails standard:fix` then we will be all set.

**Comment by jmilljr24** [comment]
@all-contributors
please add @sean-dickinson  for code.

**Review by jmilljr24 — APPROVED** [review]
lgtm!

**Pull request merged**